README.markdown

```diff
@@ -17,6 +17,13 @@ included documentation.
 Installation
 ------------
 
+If you are using VIM version 8 or higher you can use its built-in package management, just run below commands in your terminal:
```

Comment:
Let the reader know where to find more information about packages.

```suggestion
If you are using VIM version 8 or higher you can use its built-in package management; see `:help packages` for more information. Just run these commands in your terminal:
```

Reply:
I think now it is ready to merge :)